src/dfcx_scrapi/tools/nlu_util.py

```diff
@@ -129,22 +130,26 @@ def _load_data(
             flow_display_name, agent_id
         )
 
-        page_loader = pages.Pages(creds=self.creds)
-        page_map = page_loader.get_pages_map(self.flow.name, reverse=True)
-        page_id = page_map.get(page_display_name, None)
-        if page_id is None:
-            raise ValueError(
-                f'Page "{page_display_name}" does not exist in the '
-                "specified agent."
-            )
-        self.page = page_loader.get_page(page_id)
-
         trg_loader = transition_route_groups.TransitionRouteGroups(
             creds=self.creds
         )
-        self.trgs = trg_loader.list_transition_route_groups(self.flow.name)
+        agent_trgs = trg_loader.list_transition_route_groups(agent_id)
+        self.trgs = (
+            agent_trgs + trg_loader.list_transition_route_groups(self.flow.name)
+        )
         self.name_to_trg = {i.name: i for i in self.trgs}
 
+        if page_display_name != "Start Page":
```

Comment:
@hjosiah can you confirm that these changes do not break any existing functionality?

I've read through all of the code changes you've made in all files and they all seem mostly straightforward. However, in this file, there is a large chunk of code / logic shuffled around.

My assumption is that you're doing this to accommodate the use case where the user wants to inspect the Intent Similarity when `page == Start Page` and also support the "standard" functionality when `page != Start Page`

The main issue is that I cannot test this locally because `scann` is not available to be installed on Macbooks. :(

Just want to get a double confirmation from you that you've fully tested this before I merge.